src/components/Login/Login.js

```diff
@@ -0,0 +1,110 @@
+import React, { useState } from 'react'
```

Comment:
Why did you create another Login.js?

Reply:
i didnt create a new one. there was a typo i fixed when i first cloned it. Seems the login file and folder is no longer being used

Reply:
Okay. But right now on your branch, there's a login.js and a Login.js, and they have slightly different code. Maybe you should take a look at this and decide if one needs renaming or removing

Reply:
alright i'll ask Shaun or Alex which file is being used